website_product_share/README.rst

```diff
@@ -0,0 +1,54 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License
+
+Product Share
+============
+This modules brings the users the possibility to globally enable/disable a share button 
+that will be displayed in each product page.
+
+Usage
+=====
+
+1. Go to website
+2. Open one of the product
+3. Click on customize button
+4. Toggle share product option to enable/disable share button
+
+NB: even if you have to go to one specific product, the set up is valid for all of them.
+
+Know issues / Roadmap
+=====================
+
+
```

Comment:
Can you disable that button locally for a product? If not, put here this as a known issue.